### PR TITLE
Prepare for 1.9.3 release candidate

### PR DESCRIPTION
**Pull request by nielsmh**
Two major issues for the macOS version are the primary reason for this release, but there are also a few other minor fixes/improvements that can be included.

- #7657 Could not create elements on Storybook pages with ID > 255
- #7669 Decrease buttons in Cheat window not working with UI scaling
- #7686 Spelling error
- #7690 Better error message demolishing an occupied airport
- **#7705 OSX crash sorting malformed UTF-8 strings**
- #7709 Easier to make orders to industry neutral stations (this one is actually entirely rewritten)
- #7714 Highscore screen scaling
- #7627 Allow building road stops over town-owned one-way roads, instead of crashing
- #7701 & #7725 Crash in AI in multiplayer if reloaded while waiting for a command return
- **#7721 OSX colourspace conversion causing bad performance**

### CODE REVIEW

**Comment by nielsmh** [comment]
Looking at these changes a little closer, it seems that apart from #7657 and #7701 the remaining changes should be network compatible with 1.9.2, although I'm not entirely sure about the airport #7690 either.

File: os/debian/changelog
Comment (glx22) [inline os/debian/changelog]: probably 1.9.3~RC1-0 like other RCs

**Review by glx22 — DISMISSED** [review]
Seems ok to me, but maybe wait for another review

**Review by LordAro — APPROVED** [review]

**Review by glx22 — APPROVED** [review]

**Pull request merged**
